.github/workflows/node.yml

```diff
@@ -269,3 +271,45 @@ jobs:
                       node/test-report*.html
                       utils/clusters/**
                       benchmarks/results/**
+
+    test-modules:
+        if: github.event.pull_request.head.repo.owner.login == 'valkey-io'
+        # environment: AWS_ACTIONS
+        name: Running Module Tests
+        runs-on: [self-hosted, linux, ARM64]
+        timeout-minutes: 15
+
+        steps:
+            - name: Setup self-hosted runner access
+              run: sudo chown -R $USER:$USER /home/ubuntu/actions-runner/_work/valkey-glide
+
+            - uses: actions/checkout@v4
+              with:
+                  submodules: recursive
+
+            - name: Use Node.js 16.x
+              uses: actions/setup-node@v3
+              with:
+                  node-version: 16.x
+
+            - name: Build Node wrapper
+              uses: ./.github/workflows/build-node-wrapper
```

Comment:
Can you fix this step so it won't build valkey if no version given please?

Reply:
How should this be done?